src/main/java/view/SearchView.java

```diff
@@ -1,6 +1,9 @@
 package view;
 
+import data_access.movies.MovieDBDataAccessException;
+import interface_adapter.search.SearchController;
 import interface_adapter.search.SearchViewModel;
+import use_case.search.SearchByCriteriaInputData;
```

Comment:
```suggestion
import interface_adapter.search.SearchController;
import interface_adapter.search.SearchViewModel;
```
these imports are unused, also the import statement order should be fixed for checkstyle as well